postgresql-17.yaml

```diff
@@ -195,6 +200,68 @@ subpackages:
       provides:
         - libpq=${{package.full-version}}
 
+  - name: plpython3
```

Comment:
Please include the postgres major version in the name like with the other packages.